libs/sdk-core/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Allow to modify the HTTP headers in the Nuclia REST API calls.
 - Add support for `css_selector` in Link fields
+- Support POST requests for `/catalog` endpoint
```

Comment:
1.14.0 version has been released today in a previous PR but I forgot to add the date in the CHANGELOG. Can you add it and create a new `1.14.1` entry please?

Reply:
Fixed